developer/ruby/UpdateHVACLibrary.rb

```diff
@@ -2,6 +2,13 @@
 #
 # openstudio.exe UpdateHVACLibrary.rb
 
+# TODO: Julien Marrec 2019-05-23
```

Comment:
I don't like shortcutting the version translator, the real issue is why on earth this takes 2 hrs!

Reply:
The standards OSM are humongous. `212,055` lines for `DEER_2014.osm` for eg. `grep "OS:" DEER_2014.osm | wc -l` gives out 18041 objects in the file.

It would probably be a good idea to do some profiling and see if we couldn't cut the time it takes to load these models though...